opac_proc/core/assets.py

```diff
@@ -93,27 +93,49 @@ def _extract_media(self):
                 xml_et.iterfind(attrib, namespaces=namespaces)
                 for attrib in attribs
             ]
-            elements = itertools.chain(*attrib_iters)
-            medias = [
+            hrefs = [
                 element.attrib['{http://www.w3.org/1999/xlink}href']
-                for element in elements
+                for element in itertools.chain(*attrib_iters)
+            ]
+            extension_files_list = config.PROC_MEDIA_EXTENSION_FILES.split(',')
+            medias = [
+                href
+                for href in hrefs
+                if href.split('.')[-1] in extension_files_list
             ]
         else:
             parser = "html.parser"
             soup = BeautifulSoup(self.content, parser)
-            medias = [
+            src_tags = [
                 tag.get('src').strip().encode('utf-8')
                 for tag in soup.find_all(src=True)
             ]
 
+            def _is_external_link(src_tag):
+                ext_link_indicators = config.PROC_MEDIA_EXT_LINKS_IND.split(',')
+                return any(
+                    map(lambda ext_link_ind: src_tag.startswith(ext_link_ind),
+                        ext_link_indicators)
+                )
+
+            medias = [
+                src_tag
+                for src_tag in src_tags
+                if not _is_external_link(src_tag)
+            ]
+
         return medias
 
     def _get_media_path(self, name):
         """
         Returns the folder path of media.
         """
+        asset_path = config.OPAC_PROC_ASSETS_SOURCE_MEDIA_PATH
+        if name.split('.')[-1].lower() == 'pdf':
```

Comment:
Aqui uma recomendação, usar o função: `os.path.splitext` 
- docs de py2: https://docs.python.org/2/library/os.path.html#os.path.splitext
E uma breve explicação de porque é uma boa ideia usar `splitext` x o tradicional `split`
- https://stackoverflow.com/questions/542596/benefits-of-os-path-splitext-over-regular-split#542619